### PR TITLE
Support Scribbles in monailabel

### PR DESCRIPTION
**Pull request by SachidanandAlle**
This raised on behalf of @masadcv

### CODE REVIEW

**Review by aihsani — CHANGES_REQUESTED** [review]

File: monailabel/interfaces/tasks/postproc.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                ]
+
+        """
+        pass
+
+    def run_postprocessor(self, data):
+        return self.run_transforms(data, self.postproc(), log_prefix="PROC")
+
+    def pre_transforms(self):
+        return []
```
[the remark this answers is not in the document]
Comment (SachidanandAlle) [reply monailabel/interfaces/tasks/postproc.py:67]: Agreed.. just create a PostProcTask

File: sample-apps/segmentation_spleen_postproc/main.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        self.mmar = "clara_pt_spleen_ct_segmentation_1"
+
+        super().__init__(app_dir, studies, os.path.join(self.model_dir, "train_stats.json"))
+
+    def init_infers(self):
+        infers = {
+            "Spleen_Segmentation": SegmentationWithWriteLogits(
+                self.final_model, load_from_mmar(self.mmar, self.model_dir)
+            ),
+            "BIFSeg+CRF": SpleenBIFSegCRF(),
```
[the remark this answers is not in the document]
Comment (SachidanandAlle) [reply sample-apps/segmentation_spleen_postproc/main.py:36]: create a postproc task (similar to infer, strategy, scoring etc..) and init the same.. then u don't need this additional short-cut in your app to capture the logits etc..

File: monailabel/interfaces/tasks/postproc.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                    monailabel.interface.utils.Restored(keys='pred', ref_image='image'),
+                    monailabel.interface.utils.ExtremePointsd(keys='pred', result='result', points='points'),
+                    monailabel.interface.utils.BoundingBoxd(keys='pred', result='result', bbox='bbox'),
+                ]
+
+        """
+        pass
+
+    def run_postprocessor(self, data):
+        return self.run_transforms(data, self.postproc(), log_prefix="PROC")
```
[the remark this answers is not in the document]
Comment (SachidanandAlle) [reply monailabel/interfaces/tasks/postproc.py]: removed this file.. lets not have a confusion for postproc.. like deepedit/deepgrow we will have a model/infer for scribbles.. rest all logic will be same as infer.. for inference user can hook any callable methods

File: sample-apps/segmentation_spleen_postproc/main.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (SachidanandAlle) [reply sample-apps/segmentation_spleen_postproc/main.py:36]: Now scribbles is a model which has pre/infer/post actions inside.. there is no more postproc kind

**Review by aihsani — APPROVED** [review]
